### ISSUE TITLE
Ticket-Generation Modal Issues

### ISSUE THREAD

**Issue by Shivam327** [frappe/helpdesk#1178]
So, I recently implemented, Helpdesk and I have some issue. I am mentioning them below.
1) Created By Dropdown is empty (shows no results found).
![image](https://github.com/frappe/helpdesk/assets/23412311/7750d682-d163-4193-a932-251403325334)
2) Asterisk or Something should be present for fields which are compulsory and which are optional.
![image](https://github.com/frappe/helpdesk/assets/23412311/e0e838f6-e381-4328-840e-dbc4eca85660)
3) Cancel button of all the pop-up is not working. after clicking on X popup don't gets closed.
![image](https://github.com/frappe/helpdesk/assets/23412311/ce40cf65-8052-4029-8317-152c087bdbb5)

**Comment by Shivam327** [comment]
@ssiyad Any update on this issue ?? Important issue is Created By is empty. Please Look into this.

**Comment by Shivam327** [comment]
@ssiyad 

I have traced an issue in the code where a query error is occurring and resulting in an empty dropdown list in the New Ticket Creation functionality. Here is an overview of the problem and the relevant code snippets:

Query Error:
In the file [client.py](https://github.com/frappe/helpdesk/blob/main/helpdesk/extends/client.py), I identified that the method get_list is responsible for fetching the data for the dropdown list in the New Ticket Creation form. However, when the current query is executed, it throws an error, and as a result, no data is returned to the frontend. This can be observed in the first image provided in the comments.

Updated Query:
To resolve the issue, the WHERE clause in the query needs to be modified to use the table name "tabContact" instead of the current value. This change allows the query to execute successfully and retrieve the necessary data. The relevant code responsible for passing the query from the frontend can be found in [NewTicketDialog.vue](https://github.com/frappe/helpdesk/blob/main/desk/src/pages/desk/ticket-list/NewTicketDialog.vue), specifically at line 60.

To address this problem, the following actions can be taken:
Update the WHERE clause in the query to use the correct table name "tabContact" in order to retrieve the required data.


**Raw Query** 
![image](https://github.com/frappe/helpdesk/assets/23412311/3716c9ca-e306-4b64-826f-05898df66862)
**Update Query**
![image](https://github.com/frappe/helpdesk/assets/23412311/25c344b4-c0cf-4136-86c8-7ea5db5a069f)
